python/lsst/ip/diffim/subtractImages.py

```diff
@@ -126,6 +126,11 @@ class AlardLuptonSubtractConfig(lsst.pipe.base.PipelineTaskConfig,
         dtype=bool,
         default=True,
     )
+    doApplyFinalizedPsf = lsst.pex.config.Field(
+        doc="Whether to apply finalized psf models and aperture correction map.",
```

Comment:
I'm open to changing the name, though I think at least the external wcs will need special treatment to ensure that it is applied the same in this Task and `GetTemplateTask`. That inclines me to include each set of external calibrations that comes from a different input connection as its own config option.

Reply:
Ok, that's reasonable. I mostly just wanted us to make sure we thought it through. makeWarp and forcedPhot both use a separate config option for each, so this would look the same (though I question the `FinalizedPsf` vs. `ExternalWcs` naming choices). I do want to ask DRP whether that's what they want the final API to look: I'd be surprised if there's a case where you want to use one but not the other.